osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -32,6 +34,17 @@ private void load(FrameworkConfigManager config, OsuGame game)
             this.game = game;
 
             letterboxing = config.GetBindable<bool>(FrameworkSetting.Letterboxing);
+            sizeFullscreen = config.GetBindable<Size>(FrameworkSetting.SizeFullscreen);
+
+            var resolutions = getResolutions();
+            var resolutionDropdownBindable = new BindableInt(resolutions.FirstOrDefault(r => r.Key.StartsWith($"{sizeFullscreen.Value.Width}x{sizeFullscreen.Value.Height}")).Value);
+
+            resolutionDropdownBindable.ValueChanged += _ =>
```

Comment:
You should change the parameter name from `_` to something else.
The osu project normally only uses `_` as a name for unused parameters in lambdas.